tests/test_makeRawVisitInfoViaObsInfo.py

```diff
@@ -87,6 +87,8 @@ def testMakeRawVisitInfoViaObsInfo(self):
         # The header can possibly grow with header fix up provenance.
         self.assertGreaterEqual(len(self.header), beforeLength)
         self.assertEqual(visitInfo.getInstrumentLabel(), "SomeCamera")
+        # Check focusZ with default value from astro_metadata_translator
+        self.assertEqual(visitInfo.getFocusZ(), 0.0)
```

Comment:
I think it would be a better test if you added some translation code at the top (even if trivial map for FOCUSZ test header),

Reply:
Yeah, I was thinking about that. I've updated the test.